cluster/saltbase/salt/kube-node-unpacker/kube-node-unpacker.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
Is this a copy/paste of the script we use to side-load the master images? Can we share code?

I also think it would be valuable to look at trying to pull these images from gcr.io rather than side-loading them (this is adding more tech debt into our salt configs).